CONTRIBUTING.md

```diff
@@ -122,6 +123,7 @@ websites:
     hardware: Yes
     otp: Yes
     u2f: Yes
+    multipleu2f: No
```

Comment:
Please do not add tags which are marked with "No". This may be wrong elsewhere in these instructions as well. Upstream decided to leave them out, because every missing tag is evaluated as "No" anyway. This way it is more clean.

Reply:
I like it more explicit but don't super care.
Note: the `tfa` tags do have explicit `No`'s in this document and it was an attempt to follow that.